### ISSUE TITLE
Configure Google Analytics

### ISSUE THREAD

**Issue by nikdragovic** [emory-libraries/web-redesign#201]
GA tracking needs to be enabled across all pages before launch. For future-proofing, using GTM is preferred.

**Comment by kristianserrano** [comment]
Will use conventional Google Analytics instead of Google Tag Manager.

**Comment by rosy1280** [comment]
@rosy1280 needs to check with HH if there is any coordination that we need to do to make analytics work.  @kristianserrano added a google analytics field in site settings.  is that integrated into their markup?

**Comment by rosy1280** [comment]
@kristianserrano HH has not done anything to incorporate analytics into the page because it doesn't appear in the HTML in the pattern library.  We'll need to go back and add it in to each page type.

**Comment by kristianserrano** [comment]
@rosy1280 Thank you. That should be an easy thing to add.

**Comment by kristianserrano** [comment]
@rosy1280 I've added the property ID into the site in `/meta/api-ids` and shared the script with HH. I asked if they would be ok with me creating the format, or if they'd prefer to manage it on their end. Fernando indicated that he'll create the format in Cascade.

Additionally, I created a Google Tag Manager container as well. I did some cursory exploration of how to set that up. I haven't tested it with anything, but it's there if we want to investigate further and transition to it in the future. The LTDS Google account has access to it.
